packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -229,6 +227,13 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 
 		// Begin uploading stashed blobs from previous container instance
 		Object.entries(stashedBlobs).forEach(([localId, entry]) => {
+			if (entry.minTTLInSeconds && entry.localUploadTime) {
+				const timeLapseSinceLocalUpload = (Date.now() - entry.localUploadTime) / 1000;
```

Comment:
always use utc time, not local time

Reply:
I didn't understand. Date.now() uses UTC time. Do you mean the variables names?

Reply:
this was my bad. date.now always returns time since unix epoch in utc, so this should be fine